package.json

```diff
@@ -8,7 +8,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node server.js",
-    "test": "jest && npm run lint",
+    "test": "jest --detectOpenHandles --forceExit && npm run lint",
```

Comment:
Added to workaround known Jest issue: https://github.com/facebook/jest/issues/9473#issuecomment-584356203